### PR TITLE
replace logo-wide on share page as well with better icon + text

### PR DESCRIPTION
**Pull request by jancborchardt**
That was the only spot we still used the ugly logo-wide. Removed now. Please review @owncloud/designers @oparoz @schiesbn

### CODE REVIEW

**Comment by jancborchardt** [comment]
@carlaschroder on the [theming docs page](https://doc.owncloud.org/server/8.0/admin_manual/operations/theming.html?highlight=theming) the logo-wide.svg has to be replaced with logo-icon, 62*34px.

**Comment by karlitschek** [comment]
:+1:

**Comment by PVince81** [comment]
Doesn't break the main page, public page nor the documents app :+1:

**Comment by PVince81** [comment]
@owncloud-bot retest this please

**Comment by jancborchardt** [comment]
@owncloud-bot retest this please

**Comment by jancborchardt** [comment]
@DeepDiver1975 what's the deal here with CI?

**Comment by MorrisJobke** [comment]
> @DeepDiver1975 what's the deal here with CI?

They are broken:

```
00:19:22.867 1) Test_Archive_TAR::testGetFiles
00:19:22.867 cant find desktopapp.png in archive
00:19:22.867 Failed asserting that an array contains 'desktopapp.png'.
00:19:22.867 
00:19:22.867 /var/jenkins/workspace/pull-request-analyser-ng-simple@3/label/SLAVE/tests/lib/archive.php:32
00:19:22.867 
00:19:22.867 2) Test_Archive_TAR::testExtract
00:19:22.867 Failed asserting that false matches expected true.
00:19:22.867 
00:19:22.867 /var/jenkins/workspace/pull-request-analyser-ng-simple@3/label/SLAVE/tests/lib/archive.php:116
00:19:22.867 
00:19:22.867 3) Test_Archive_ZIP::testGetFiles
00:19:22.867 cant find desktopapp.png in archive
00:19:22.867 Failed asserting that an array contains 'desktopapp.png'.
00:19:22.867 
00:19:22.867 /var/jenkins/workspace/pull-request-analyser-ng-simple@3/label/SLAVE/tests/lib/archive.php:32
00:19:22.867 
00:19:22.867 4) Test_Archive_ZIP::testExtract
00:19:22.868 Failed asserting that false matches expected true.
00:19:22.868 
00:19:22.868 /var/jenkins/workspace/pull-request-analyser-ng-simple@3/label/SLAVE/tests/lib/archive.php:116
```

**Comment by MorrisJobke** [comment]
@jancborchardt You need to fix them.

**Comment by carlaschroder** [comment]
@jospoortvliet This might interest you.

**Comment by jospoortvliet** [comment]
@carlaschroder what, that Jan broke something? ;-)

**Comment by carlaschroder** [comment]
@jospoortvliet Apparently :)

**Comment by jancborchardt** [comment]
@MorrisJobke @PVince81 can you check those out? Not sure if the tests need to be changed, or the file in the zip (or tarfile ...?). :\

File: tests/lib/archive.php [hunk 1]
```diff
@@ -26,7 +26,7 @@
 	public function testGetFiles() {
 		$this->instance=$this->getExisting();
 		$allFiles=$this->instance->getFiles();
-		$expected=array('lorem.txt','logo-wide.png','dir/', 'dir/lorem.txt');
```
Comment (DeepDiver1975) [inline tests/lib/archive.php]: @jancborchardt revert the changes in this file and the unit test shall pass again

Comment (jancborchardt) [inline tests/lib/archive.php]: @DeepDiver1975 thanks!

**Comment by jancborchardt** [comment]
Done, final test please @MorrisJobke :)

@owncloud/designers also make sure to review https://github.com/owncloud/gallery/issues/189 and https://github.com/owncloud/galleryplus/pull/153

**Comment by MorrisJobke** [comment]
Works :+1:
